# ENUM: 'HsvColorDetector',
#       'PixelListColorDetector',
#       'DynamicPixelListColorDetector'
color_detector: 'HsvColorDetector'
debug_printer_classes: ''

# ROS:
# Subscriber
ROS_img_msg_topic: 'image_raw'
ROS_img_queue_size: 1
# Publisher
ball_msg_topic: 'ball_in_image'
line_msg_topic: 'line_in_image'

# HsvColorDetector:
HSV_lower_values_h: 0)
HSV_lower_values_s: 0)
HSV_lower_values_v: 0)
HSV_upper_values_h: 255)
HSV_upper_values_s: 255)
HSV_upper_values_v: 255)

# PixelListColorDetector:
pixel_list_color_space_path: '/config/testfield_color01.yaml'
